src/me/fru1t/subedit/utils/Utils.kt

```diff
@@ -7,52 +7,120 @@ import java.io.File
  * multiple tools.
  */
 object Utils {
-  private const val NEWLINE = "\r\n"
+  public const val SUB_NEWLINE = "\\N"
 
-  /** Prompts the user for a file. Returns the file if valid or `null` if no file could be prodded from the user. */
-  fun askForFile(): File? {
+  private val DIALOGUE_REGEX = Regex("^Dialogue: ([^,]+),([^,]+),([^,]+),([^,]+),([^,]*),(\\d+),(\\d+),(\\d+),([^,]*),(.*)\$")
+  private val COLOR_REGEX = Regex("[0-9A-F]{6}")
+  private val ALPHA_REGEX = Regex("[0-9A-F]{2}")
+
+  /** Prompts the user for a file to read from.
+   * Returns the file if valid or `null` if no file could be prodded from the user. */
+  fun askForSourceFile(): File? {
     println("File path?")
     val file = File(readLine() ?: return null)
 
     if (!file.exists()) {
       println("Didn't find file... Try again?")
       if (askYes()) {
-        return askForFile()
+        return askForSourceFile()
       }
       return null
     }
 
-    if (!file.canWrite() || !file.canRead()) {
+    if (!file.canRead()) {
       println("Looks like it's protected from here, sorry")
       return null
     }
 
     return file
   }
 
+  /** Prompts the user for a file to output to.
+   * Returns the file if valid or `null` if no file could be prodded from the user. */
```

Comment:
Multi-line kdoc should have the following format:

```
/**
 * First line of comments here.
 */ 
```

Reply:
done!

Reply:
Almost! ending `*/` should be on a new line too